src/main/java/org/folio/innreach/domain/service/impl/HoldingsServiceImpl.java

```diff
@@ -34,6 +36,13 @@ public Optional<Holding> find(UUID holdingId) {
     return holdingsStorageClient.findHolding(holdingId);
   }
 
+  @Override
+  public void delete(UUID holdingId) {
+      holdingsStorageClient.findHolding(holdingId)
+        .ifPresentOrElse(holding -> holdingsStorageClient.deleteHolding(holdingId)
+          ,()->log.info("Holding not found with holdingId:{}",holdingId));
```

Comment:
Think you want spaces after the commas. Also a space after the `->`. Maybe put the comma that is at the beginning of the line at the end of the hline above.

Reply:
done